code/game/machinery/computer/_computer.dm

```diff
@@ -26,11 +26,8 @@
 
 /obj/machinery/computer/Initialize(mapload, obj/item/circuitboard/C)
 	. = ..()
+
 	power_change()
-	if(!QDELETED(C))
-		qdel(circuit)
-		circuit = C
-		C.moveToNullspace()
```

Comment:
shouldn't this move C into the computer and assign it to circuit? or we should remove the argument if it's unused?

Reply:
yeah, no idea why TG kept it if its unused

Reply:
resolved in b79cded